### PR TITLE
Bump harvester-load-balancer to v0.2.5

### PR DESCRIPTION
**Pull request by w13915984028**
To fix https://github.com/harvester/harvester/issues/5137, the harvester-load-balancer image has a new v0.2.5 release.

Bump a new chart v0.2.5 release to Harvester.

### CODE REVIEW

**Comment by bk201** [comment]
Yes, you are correct. If a chart doesn't change, we just need a PR to bump the image tag in release branches: https://github.com/harvester/harvester/blob/f8469881441b92a71c6e58a493f433eccf5f3ba1/deploy/charts/harvester/values.yaml#L458, can you help create a PR for it? Thanks.

Bumping a chart when its image changes is also OK, but it is only useful when you use the chart without harvester or for testing/dev purposes.

File: charts/harvester-load-balancer/values.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ image:
   repository: rancher/harvester-load-balancer
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: v0.2.0
+  tag: v0.2.5
```
Comment (starbops) [inline charts/harvester-load-balancer/values.yaml:11]: The image tag defaults to the chart appVersion, so we can drop the explicit version string unless we need something else here.

```suggestion
  tag: ""
```

**Review by starbops — APPROVED** [review]
Just one nit, else LGTM. Thank you.

**Comment by w13915984028** [comment]
The `harvester-load-balancer` chart is not used out of Harvester cluster. Instead of releasing a new chart version, just bump the image tag in Harvester v1.3 branch. 

This PR is replaced by https://github.com/harvester/harvester/pull/5354

**Pull request closed without merging**